src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -16,10 +16,11 @@ object ResultView {
     fun printReturn(lottoResult: LottoResult) {
         println("당첨 통계")
         println("---------")
-        println("3개 일치 (${LottoReturn.FOURTH.returnPrice}원)- ${lottoResult.fourthCount}개")
-        println("4개 일치 (${LottoReturn.THIRD.returnPrice}원)- ${lottoResult.thirdCount}개")
-        println("5개 일치 (${LottoReturn.SECOND.returnPrice}원)- ${lottoResult.secondCount}개")
-        println("6개 일치 (${LottoReturn.FIRST.returnPrice}원)- ${lottoResult.firstCount}개")
+        println("3개 일치 (${LottoReturn.FIFTH.returnPrice}원)- ${lottoResult.count(LottoReturn.FIFTH)}개")
```

Comment:
```suggestion
        println("${LottoReturn.FIFTH.matchCount}개 일치 (${LottoReturn.FIFTH.returnPrice}원)- ${lottoResult.count(LottoReturn.FIFTH)}개")
```
만약 요구사항이 변경되어 매칭되는 숫자 값이 변경되더라도 출력에는 변경이 없도록 개선해보면 어떨까요?